src/lib/wallet/index.ts

```diff
@@ -101,18 +101,15 @@ export default class Wallet extends Web3Wallet implements HederaNativeWallet {
     })
   }
 
-  /*
-   *  Session Requests
-   */
   public parseSessionRequest(
```

Comment:
I think we could use a recursive function for the first param for all of these JSON-RPC methods to test whether or not it's an array, and if so return an array of bodies

Reply:
we'd then update the <parse|execute>SessionRequest functions to support this and theoretically, we'd be able able to pass along multiple requests of same method type without making to many modifications to parts of the code